localisation/english/btr_technologies_l_english.yml

```diff
@@ -57,3 +57,9 @@
  btr_tech_sysall_hahne:0 "Hahne-Kedar"
  btr_tech_sysall_hahne_desc:0 "Reliable and efficient, Earth-based Hahne-Kedar has become a major supplier to the Systems Alliance military. Their weapons are considered stock quality at best, though their armor lines are generally recognized as above average when compared to the rest of the citadel races."
  
+### Parnack Regime Technology
+
+btr_tech_parnkreg_regAlpha:0 "Registry of Alphas"
+btr_tech_parnkreg_regAlpha_desc:0 "With the new technological inventions we are able to store all recorded and oral information on the biggest alphas in our story. This "Registry of Alphas" will serve at the same time as a political guide for our leaders, a historical treasure for our intellectuals and cultural entertainment and inspiration for our people."
+btr_tech_parnkreg_standArm:0 "Standing Armies"
+btr_tech_parnkreg_standArm_desc:0 "While our society is fractioned by packs by evolutionary instinct, progress and science are making social divisions based on packs less important by the day. Yahg are starting to see their whole species as one big pack, confronted by untold hostile aliens packs. In light of this, a standing, professional, non-tribal army is starting to look feasible as long as our people focus their inherent aggressiveness on other species."
```

Comment:
While our society is fractured by packs by evolutionary instinct, progress and science are making pack based social divisions less important by the day. Yahg are starting to see their whole species as one cohesive pack, confronted by untold hostile aliens packs. In light of this, a standing, professional, non-tribal army is starting to look feasible as long as our people focus their inherent aggressiveness on other species.

Reply:
Fixed